app/controllers/parks_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::ParksController < ApplicationController
```

Comment:
This file isn't an API controller! Also, since you're using React, you don't need lines 4 or 8 -- the only purpose of these actions is to eventually point to a view with a `<div id='app' />`

Reply:
Updated, thanks for the direction!